tests/conftest.py

```diff
@@ -249,7 +233,7 @@ def _trigger_experiment_loader():
 def fixture_check_ping_for_experiment(trigger_experiment_loader):
     def _check_ping_for_experiment(experiment=None):
         control = True
-        timeout = time.time() + 60 * 5
+        timeout = time.time() + 60 * 2
```

Comment:
Am i reading this right, the timeout dropped from 5m to 2min?

Reply:
Yeah, I could probably change this to just 1 min.

Reply:
Actually, that fixture can be removed.